### DISCUSSION TITLE
How to catch $Request $post From Normal Forms

### DISCUSSION THREAD

**Discussion by pratikraman-dev** [orchidsoftware/platform#2321] [Q&A]
i have created my normal forms and i want to catch form in $request how can i achieve that in orchid screen ??

if anyone please help me out

**Comment by tabuna** [comment]
It is difficult for me to answer this question, since it does not involve any code. DI works for screen methods, just like controllers. That is, you can get your own object:

```php
/**
 * @param Request $request
 */
public function sendMessage(Request $request)
{
    // code...
}
```
